docs/quick_start/README.md

```diff
@@ -45,6 +50,16 @@ We need to modify the Docker Daemon configuration to use the Dragonfly as a pull
 systemctl restart docker
 ```
 
+For macOS users:
```

Comment:
The doc of `quick_start` should be as concise as possible. We treat Linux as default and maybe you can add an external link for MacOS or something else. WDYT?

Reply:
Yes, I will remove the part of macos.